src/main/java/org/elasticsearch/cluster/block/ClusterBlockLevel.java

```diff
@@ -29,9 +30,19 @@
 public enum ClusterBlockLevel {
     READ(0),
     WRITE(1),
-    METADATA(2);
 
-    public static final EnumSet<ClusterBlockLevel> ALL = EnumSet.of(READ, WRITE, METADATA);
+    // todo-tlrx Update version number according to the branch it is merged to
```

Comment:
you migth want to use //nocommit so maven notifies you when you forget about it ;)